README.md

```diff
@@ -36,25 +42,11 @@ The earlier you remediate a vulnerability in the release cycle, the lower the co
 or even sooner, during the development.
 
 The JFrog VS Code Extension adds JFrog Xray scanning of project dependencies to your VS Code IDE.
-It allows developers to view panels displaying vulnerability information about the components and their dependencies directly in their VS Code IDE.
-With this information, a developer can make an informed decision on whether to use a component or not before it gets entrenched into the organization’s product.
+It allows developers to view panels displaying vulnerability information about the components and their dependencies directly in their VS Code IDE. The information can be both about the current project developed, and about builds on the CI system.
 
```

Comment:
Let's add the following paragraph here - 

The extension also allows developers to track the status of the code while it is being built, tested and scanned on the CI server.